### PR TITLE
Add SECURITY.md

### PR DESCRIPTION
**Pull request by NSeydoux**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Security policies
```
Comment (matthieubosquet) [inline SECURITY.md:1]:
```suggestion
# Security policy
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+This library intends supporting the development of Solid applications reading and
+writing data in user's Pods. User data should always be considered sensitive and
```
Comment (matthieubosquet) [inline SECURITY.md:4]:
```suggestion
writing data in Solid servers. Data should always be considered sensitive and
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+be processed carefully, in particular private data which is obtained through
```
Comment (matthieubosquet) [inline SECURITY.md:5]:
```suggestion
be processed with care and regards to access restrictions and personal information.
```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+authentication.
```
Comment (matthieubosquet) [inline SECURITY.md:6]:
```suggestion
```

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+For a better separation of concerns, this library does not deal directly with
+authentication. In order to make authenticated requests, one should inject a `fetch`
+function compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters)
+dealing with authentication. This may be done using Inrupt's authentication libraries
+[for Node](https://www.npmjs.com/package/@inrupt/solid-client-authn-node) or [for
+the browser](https://www.npmjs.com/package/@inrupt/solid-client-authn-browser).
```
Comment (matthieubosquet) [inline SECURITY.md:13]: Maybe insert a see also to solid client authn’s security policy.

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+This library also exposes functions to modify access permissions to user data. We
+strive to make the API and documentation as clear and intuitive as possible, because
+misusing these functions may result in exposing user data beyond what is intended.
+Please do open [issues](https://github.com/inrupt/solid-client-js/issues) if you
+faced difficulties in that area.
```
Comment (matthieubosquet) [inline SECURITY.md:19]:
```suggestion
face difficulties in that area. 
```

File: SECURITY.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+authentication. In order to make authenticated requests, one should inject a `fetch`
+function compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters)
+dealing with authentication. This may be done using Inrupt's authentication libraries
+[for Node](https://www.npmjs.com/package/@inrupt/solid-client-authn-node) or [for
+the browser](https://www.npmjs.com/package/@inrupt/solid-client-authn-browser).
+
+This library also exposes functions to modify access permissions to user data. We
+strive to make the API and documentation as clear and intuitive as possible, because
+misusing these functions may result in exposing user data beyond what is intended.
+Please do open [issues](https://github.com/inrupt/solid-client-js/issues) if you
```
Comment (matthieubosquet) [inline SECURITY.md:18]:
```suggestion
Please do open an [issue](https://github.com/inrupt/solid-client-js/issues) if you
```

File: SECURITY.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+For a better separation of concerns, this library does not deal directly with
+authentication. In order to make authenticated requests, one should inject a `fetch`
+function compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters)
+dealing with authentication. This may be done using Inrupt's authentication libraries
+[for Node](https://www.npmjs.com/package/@inrupt/solid-client-authn-node) or [for
+the browser](https://www.npmjs.com/package/@inrupt/solid-client-authn-browser).
+
+This library also exposes functions to modify access permissions to user data. We
+strive to make the API and documentation as clear and intuitive as possible, because
+misusing these functions may result in exposing user data beyond what is intended.
```
Comment (matthieubosquet) [inline SECURITY.md:17]:
```suggestion
misuse of these functions may result in exposing data beyond what is intended.
```

File: SECURITY.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+authentication.
+
+For a better separation of concerns, this library does not deal directly with
+authentication. In order to make authenticated requests, one should inject a `fetch`
+function compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters)
+dealing with authentication. This may be done using Inrupt's authentication libraries
+[for Node](https://www.npmjs.com/package/@inrupt/solid-client-authn-node) or [for
+the browser](https://www.npmjs.com/package/@inrupt/solid-client-authn-browser).
+
+This library also exposes functions to modify access permissions to user data. We
```
Comment (matthieubosquet) [inline SECURITY.md:15]:
```suggestion
This library also exposes functions to modify data access permissions. We
```

File: SECURITY.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+strive to make the API and documentation as clear and intuitive as possible, because
```
Comment (matthieubosquet) [inline SECURITY.md:16]: Maybe a link to the docs?

File: SECURITY.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+misusing these functions may result in exposing user data beyond what is intended.
+Please do open [issues](https://github.com/inrupt/solid-client-js/issues) if you
+faced difficulties in that area. 
+
+# Reporting a vulnerability
+
+If you discover a vulnerability in our code, or experience a bug related to security,
+please report it following the instructions provided on our [security page](https://inrupt.com/security/).
```
Comment (matthieubosquet) [inline SECURITY.md:24]:
```suggestion
please report it following the instructions provided on [Inrupt’s security page](https://inrupt.com/security/).
```

**Review by matthieubosquet — APPROVED** [review]
Suggested some edits, especially in the language used to describe permissions and data.
But looks good!

File: SECURITY.md (hunk 5 above)
Comment (NSeydoux) [reply SECURITY.md:12]: fd30e97f57207ac953f289b2530b4131dadf7e52

File: SECURITY.md (hunk 10 above)
Comment (NSeydoux) [reply SECURITY.md]: fd30e97f57207ac953f289b2530b4131dadf7e52

**Review by ajacksified — APPROVED** [review]

File: SECURITY.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+[for Node](https://www.npmjs.com/package/@inrupt/solid-client-authn-node) or [for
+the browser](https://www.npmjs.com/package/@inrupt/solid-client-authn-browser).
+The security policy for these libraries is available in the associated [GitHub repository](https://github.com/inrupt/solid-client-authn-js/blob/main/SECURITY.md).
+
+This library also exposes functions to modify data access permissions. We
+strive to make the API and [documentation](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/manage-access/)
+as clear and intuitive as possible, because misuse of these functions may result
+in exposing data beyond what is intended. Please do open an
+[issue](https://github.com/inrupt/solid-client-js/issues) if you face difficulties
+in that area.
```
Comment (matthieubosquet) [inline SECURITY.md:20]:
```suggestion
[issue](https://github.com/inrupt/solid-client-js/issues) if you face difficulties
in that area.
```
```suggestion
[issue](https://github.com/inrupt/solid-client-js/issues) if you face difficulties
with our access control APIs.
```

Comment (matthieubosquet) [reply SECURITY.md:20]: Sorry for the late comment @NSeydoux, but I think it would be good to be specific here.

**Pull request merged**